WooCommerce/WooCommerceUITests/Tests/OrdersTests.swift

```diff
@@ -28,6 +28,7 @@ final class OrdersTests: XCTestCase {
     }
 
     func test_create_new_order() throws {
```

Comment:
👋 @joshheald!

This test seems to be failing for a valid reason. The Add Shipping screen is only opened when the bottom sheet is expanded (please check the video below), so the test is tapping the Add Shipping button but fails since the screen is not displayed. I don't think this was the desired experience?

<details><summary>Video</summary>
<video width="300" src="https://github.com/woocommerce/woocommerce-ios/assets/42008628/1985a26d-932a-4f1b-ad8c-324a5e09831a">
</details>

Reply:
@tiagomar thabk you for the fix and the review! The Add Shipping issue is a bug, thank you for pointing it out. I found and fixed it separately, but I didn't think about these tests when I did it. See 28de986 on the next PR #11503.

Lesson learned; check that the thing actually works before assuming the test is at fault 😬